piptools/scripts/sync.py

```diff
@@ -28,6 +28,7 @@
     help="Only show what would happen, don't change anything",
 )
 @click.option("--force", is_flag=True, help="Proceed even if conflicts are found")
+@click.option("--no-deps", is_flag=True, help="Don't install package dependencies")
```

Comment:
```suggestion
@click.option("--no-deps", is_flag=False, help="Don't install package dependencies")
```
`is_flag=True` might be a breaking change.

Reply:
The `is_flag` has different meaning and value is `False` by default . Sorry, this always confuses me :(